integration-cli/docker_cli_plugins_test.go

```diff
@@ -181,7 +181,7 @@ func (s *DockerRegistrySuite) TestPluginInstallImage(c *check.C) {
 
 	out, _, err := dockerCmdWithError("plugin", "install", repoName)
 	c.Assert(err, checker.NotNil)
-	c.Assert(out, checker.Contains, "target is image")
+	c.Assert(out, checker.Contains, `Encountered MediaType "application/vnd.docker.container.image.v1+json" when fetching "image"`)
```

Comment:
@stevvooe yea that is kind of funky - I tweaked it - see if this new one is any better.